h/presenters/group_json.py

```diff
@@ -39,10 +39,6 @@ def _model(self):
 
     def _inject_urls(self, model):
```

Comment:
This little method now does so little that you might consider getting rid of it and setting `model['links'] = self.resource.links or {}` in the `asdict` method. That is, this function no longer has a real function :).

Reply:
Sounds good! Will do and merge and then do a client release.

Reply:
Please do make sure to release the client changes before these changes!

Reply:
Yes! Definitely. I'm keenly aware and nervous about making a mistake like that!